### PR TITLE
Update people_currentcrew_generalmembers.json

### PR DESCRIPTION
**Pull request by wjiang42**
Updated member list to reflect current spreadsheet (thank you Grace!)

### CODE REVIEW

File: _data/people_currentcrew_generalmembers.json [hunk 1]
```diff
@@ -6,11 +6,7 @@
     {
         "first_name": "Alice",
         "last_name": "Ho"
-    },
-    {
-        "first_name": "Mason",
-        "last_name": "Xiao"
-    },
+    }
```
Comment (DaAwesomeP) [inline _data/people_currentcrew_generalmembers.json:9]: You may need a comma here!

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Nearly got it! One small change needed for it to build the website.

File: _data/people_currentcrew_generalmembers.json (hunk 1 above)
Comment (wjiang42) [reply _data/people_currentcrew_generalmembers.json:9]: Uhh I just realised my commit overrode all the changes I did before so this pull request should probably be deleted

Comment (wjiang42) [reply _data/people_currentcrew_generalmembers.json:9]: There isn't an undo button anywhere is there?

Comment (DaAwesomeP) [reply _data/people_currentcrew_generalmembers.json:9]: Feel free to close it or push more commits!

Comment (wjiang42) [reply _data/people_currentcrew_generalmembers.json:9]: Ok I realised what I did, closing pull request

**Pull request closed without merging**